packages/@aws-cdk/custom-resources/lib/aws-custom-resource/runtime/index.ts

```diff
@@ -67,6 +67,10 @@ export async function handler(event: AWSLambda.CloudFormationCustomResourceEvent
         region: call.region,
       });
 
+      if (event.RequestType === 'Delete' && call.physicalResourceIdPath) {
+        call.parameters[call.physicalResourceIdPath] = physicalResourceId;
+      }
```

Comment:
If I understand correctly you want to use the "current" physical id somewhere in your API call?

The `physicalResourceIdPath` works the other way around: it extracts the physical id from the response and returns it to CloudFormation. You haven't updated the documentation for this prop so only you would know about this behavior here.

How about having some sort of tag (`{{PHYSICAL_ID}}`?) that could be used in the `parameters` and would get replaced in the Lambda function before making the API call?

Can you detail your specific use case maybe?

@eladb opinion on this?